course-v2/layouts/partials/course_banner.html

```diff
@@ -3,7 +3,7 @@
 <div id="course-banner" class="p-0">
   <div class="max-content-width course-banner-content m-auto py-3">
     {{ $courseData := .Site.Data.course }}
-    <span class="course-number-term-detail">{{ $courseData.primary_course_number }} | {{ $courseData.term }} {{ $courseData.year }} | {{ delimit $courseData.level ", " }}</span>
+    <span class="course-number-term-detail">{{ $courseData.primary_course_number }} | {{ $courseData.term }} {{ $courseData.year }} | {{if $courseData.level }}{{ delimit $courseData.level ", " }}{{ end }}</span>
```

Comment:
```suggestion
    <span class="course-number-term-detail">{{ $courseData.primary_course_number }} | {{ $courseData.term }} {{ $courseData.year }} {{ if $courseData.level }} | {{ delimit $courseData.level ", " }}{{ end }}</span>
```

If there is no value, I think we should prevent appending a `|` at the end.

Also, even though we don't have a defined format for mustaches, let's keep our spacing consistent with the rest of the code.